File: Server/Blog/Components/BlogML/Enumerators/ContentTypes.vb
Imports System.Xml.Serialization

Namespace BlogML
 Public Enum ContentTypes As Short
  <XmlEnumAttribute("html")> _
  Html = 1
  <XmlEnumAttribute("xhtml")> _
  Xhtml = 2
  <XmlEnumAttribute("text")> _
  Text = 3
  <XmlEnumAttribute("base64")> _
  Base64 = 4
 End Enum
End Namespace

File: Server/Core/BlogML/Enumerators/ContentTypes.cs
using System.Xml.Serialization;

namespace DotNetNuke.Modules.Blog.Core.BlogML
{
  public enum ContentTypes : short
  {
    [XmlEnum("html")]
    Html = 1,
    [XmlEnum("xhtml")]
    Xhtml = 2,
    [XmlEnum("text")]
    Text = 3,
    [XmlEnum("base64")]
    Base64 = 4
  }
}